spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/tomcat/TomcatWebServer.java

```diff
@@ -105,7 +105,8 @@ public TomcatWebServer(Tomcat tomcat, boolean autoStart, Shutdown shutdown) {
 	}
 
 	private void initialize() throws WebServerException {
-		logger.info("Tomcat initialized with port(s): " + getPortsDescription(false));
+		logger.info("Tomcat initialized with port" + (this.tomcat.getService().findConnectors().length == 1 ? "" : "s")
+				+ ": " + getPortsDescription(false));
```

Comment:
Tomcat's the only case where we use a `:`. As part of this, it would be good to make that consistent. I'm inclined to remove it from Tomcat's log message rather than adding one to the other servers' messages.

Reply:
Thanks for the feedback, kindly have another look.